modules/nf-core/ensemblvep/download/ensemblvep-download.diff

```diff
@@ -0,0 +1,59 @@
+Changes in module 'nf-core/ensemblvep/download'
+--- /dev/null
++++ modules/nf-core/ensemblvep/download/nextflow.config
+@@ -0,0 +1,20 @@
++/*
++~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
++    annotation-cache/downloadvepcache ENSEMBLVEP_DOWNLOAD base config file
```

Comment:
Why the process config within the module and not in modules.conf?

Reply:
not nf-core, so I'm shipping modules with config files to have all in one place